debian/control

```diff
@@ -1,9 +1,9 @@
 Package: dokku
-Version: 0.3.12
+Version: 0.4.3
 Section: base
 Priority: optional
 Architecture: amd64
-Depends: locales, git, make, curl, software-properties-common, docker-engine | docker-engine-cs | lxc-docker, gcc, python-software-properties, man-db, herokuish, sshcommand
```

Comment:
Can you re-add `lxc-docker`? It's just 1.6.2, so if we make it at least that version, then we should be fine, right? I'd rather not break systems that haven't switched their remote just yet.